.github/workflows/poddefaults_intergration_test.yaml

```diff
@@ -26,7 +42,7 @@ jobs:
     - name: Install cert-manager
       run: |
         echo "Installing cert-manager ..."
-        CERT_MANAGER_VERSION="v1.5.0"
+        CERT_MANAGER_VERSION="v1.10.1"
```

Comment:
@apo-ger could you convert this script into a `.sh` file and place it under https://github.com/kubeflow/kubeflow/tree/master/components/testing/gh-actions?

We might need to reuse this for other components, i.e. https://github.com/kubeflow/kubeflow/pull/6734

Reply:
done!